apps/gbmusic/app.js

```diff
@@ -433,15 +383,21 @@ function sendCommand(command) {
   drawControls();
 }
 
+function handleTouch(btn, pos) {
+  if (pos === undefined || pos.y >= Bangle.appRect.y) {
+    togglePlay();
+  }
+}
+
 function togglePlay() {
-  sendCommand(stat==="play" ? "pause" : "play");
+  sendCommand(stat==="play" ? "pause" : "playpause");
```

Comment:
Is it better to use `"playpause"` than `"play"`, I believe that's the reason for the change?

I think this goes to `.../GBMusicControlReceiver.java` in gadgetbridge, am I correct?

"playpause" should maybe be added to possible value for the `command` parameter for the `sendCommand` function?